modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectFieldImpl.java

```diff
@@ -46,6 +50,21 @@ public List<ObjectFieldSetting> getObjectFieldSettings() {
 		return _objectFieldSettings;
 	}
 
+	@Override
+	public boolean hasUniqueValues() {
+		if (compareBusinessType(
+				ObjectFieldConstants.BUSINESS_TYPE_AUTO_INCREMENT) ||
+			GetterUtil.getBoolean(
+				ObjectFieldSettingUtil.getValue(
+					ObjectFieldSettingConstants.NAME_UNIQUE_VALUES,
```

Comment:
@gabrielwas I didn't use `ObjectFieldSettings` in this case because of some strange behaviors, for example: To use settings, I would have to put this property in the `AutoIncrementObjectFieldBusinessType.getAllowedObjectFieldSettingsNames()` method, which would make it possible for the user to send a value to this property (via POST/PUT), however this value would always be ignored by us. 
I also found other bugs so due to the urgency of this PR, I thought it would be better to do it this way.

Reply:
ok!